examples/install_requirements.py

```diff
@@ -134,7 +134,7 @@ def hasWhitespace(string):
         # Get package version if in git context
         final_version = find_version.get_package_dev_version(git_commit)
         # Install latest built wheels from artifactory (0.0.0.0+[hash] or [version]+[hash])
-        commands = [[*pip_package_install, "--extra-index-url", ARTIFACTORY_URL, "depthai=="+final_version],
+        commands = [[*pip_package_install, "--extra-index-url", ARTIFACTORY_URL, "depthai"],
```

Comment:
Sorry missed this earlier. I think we want the dev0+commit version here right?

Reply:
ah was bound to happen :)